data/org.freedesktop.portal.ProxyResolver.xml

```diff
@@ -41,11 +46,53 @@
         proxy uri are of the form ``protocol://[user[:password] AT host:port``.
         The protocol can be ``http``, ``rtsp``, ``socks`` or another proxying
         protocol. ``direct://`` is used when no proxy is needed.
+
+        This is a convenience wrapper over reading the PAC file or the other
+        portal settings, in that order of priority from highest to lowest.
     -->
     <method name="Lookup">
       <arg type="s" name="uri" direction="in"/>
       <arg type="as" name="proxies" direction="out"/>
     </method>
+    <!--
+    The following values are valid:
+      0: Proxy should not be used.
+      1: Automatic proxy mode; use pac_url.
+      2: Manual proxy mode; read associated settings manually.
+
+      This property was added in version 2.
+    -->
+    <property name="use_proxy" type="u" access="read"/>
+    <!--
+      This property was added in version 2.
+    -->
+    <property name="pac_url" type="s" access="read"/>
+    <!--
+      This property was added in version 2.
+    -->
+    <property name="https_proxy" type="as" access="read"/>
```

Comment:
Why is this an array? Same question for: http_proxy, socks_proxy, and ftp_proxy. The underlying GNOME settings are not arrays. Is your expectation that apps will try one and fall back to the next if it fails? I think applications won't be prepared to do this.